src/Lykke.Job.BlockchainCashinDetector/Modules/CqrsModule.cs

```diff
@@ -100,9 +101,15 @@ private CqrsEngine CreateEngine(IComponentContext ctx, IMessagingEngine messagin
                 Register.BoundedContext(Self)
                     .FailedCommandRetryDelay(defaultRetryDelay)
 
-                    .ListeningCommands(typeof(EnrollToMatchingEngineCommand))
+                    .ListeningCommands(typeof(LockDepositWalletCommand))
```

Comment:
Changing `EnrollToMatchingEngineCommand` by `LockDepositWalletCommand` makes reader to think here's a different process now. Is it the desired effect?

Reply:
Yes, process is different now